configure.py

```diff
@@ -97,6 +97,9 @@ def is_solaris(self):
     def is_aix(self):
         return self._platform == 'aix'
 
+    def is_aix_os400_variant(self):
```

Comment:
`is_pase` would be a better name, I suspect

Reply:
Want to keep os400, so a compromise of is_os400_pase() was chosen